### DISCUSSION TITLE
Saving a file to GitHub

### DISCUSSION THREAD

**Discussion by NicolaHearn** [jgraph/drawio#3186] [Q&A]
I'm using draw.io for the first time. I've installed and authorised it on GitHub, clicked on the link to create a new file, I see a list of my GitHub repos and then I can't click "ok". Does anyone know why this is or what to try to resolve it? Thanks!
<img width="1440" alt="Screenshot 2022-11-17 at 11 40 21" src="https://user-images.githubusercontent.com/72850665/202438585-e154816a-8b1f-4009-9aec-1292aa12d874.png">

**Comment by davidjgraph** [comment]
Do you have a different browser you could test with?
